cobc/parser.y

```diff
@@ -3180,6 +3181,7 @@ set_record_size (cb_tree min, cb_tree max)
 %token V
 %token VALID
 %token VALIDATE
+%token VAL_STATUS		"VAL-STATUS"
```

Comment:
Was this a partial-merge? The token seems to be neither used in the parser, nor in the reserved word list and isn't in Changelog either.

Reply:
I can not tell, though there is a single mention of `VAL_STATUS` in [https://sourceforge.net/p/gnucobol/code/4794](https://sourceforge.net/p/gnucobol/code/4794)

Reply:
Interestingly, even at the latest revision of `branches/gnucobol-3.x`, this is the only occurrence of `VAL_STATUS`.

Reply:
Just checked: it isn't used as this is aliased in reserved.c to `VALIDATE_STATUS` as it should be. I suggest to drop the line here and whenever we change parser.y in 3.x we'll drop it there, too.

Reply:
Done.